index.src.html

```diff
@@ -17,96 +17,99 @@
+    consent. The user agent mediates access to credentials in order to preserve user privacy. Authenticators provide cryptographic proof of their properties to relying parties via attestation. This specification also describes the functional
+    model for WebAuthn conformant authenticators, including their signature and attestation functionality.
 Boilerplate: omit conformance, omit feedback-header
 Markup Shorthands: css off, markdown on
 </pre>
 
 
-# Use Cases # {#use-cases}
+# Introduction # {#intro}
 
 [INFORMATIVE]
 
-This specification defines an API for web pages to access scoped credentials through JavaScript, for the purpose of strongly
-authenticating a user. Scoped credentials are always scoped to a single <a>[RP]</a>. This scoping is enforced jointly by the
-User Agent implementing the Web Authentication API and the authenticator that holds the credential, by constraining the
-availability and usage of credentials. Scoped credentials created by a [RP] can only be accessed by web origins belonging to that
-[RP]. Additionally, privacy across [RPS] must be maintained; scripts must not be able to detect any properties, or even the
-existence, of scoped credentials belonging to other [RPS].
+This specification defines a Web API enabling the creation and use of strong, attested, cryptographic, <em><a>scoped credentials</a></em> by web applications, 
+for the purpose of strongly authenticating users. 
+A <a>scoped credential</a>, created and stored, subject to <em><a>user verification</a></em>, by an <em><a>authenticator</a></em> at the behest of a <em><a>[RP]</a></em>, can only be accessed by web origins belonging to that [RP]. 
+This scoping is enforced jointly by <em><a>conforming User Agents</a></em> and <em><a>authenticators</a></em>. 
+Additionally, privacy across <a>[RPS]</a> is maintained; [RPS]' browser scripts are not able to detect any properties, or even the existence, of credentials scoped to other [RPS].
 
-Scoped credentials are located on <a>authenticators</a>, which can use them to perform operations subject to user consent.
-Broadly, authenticators are of two types:
+[RPS] employ the <a>Web Authentication API</a> during two distinct, but related, <a>ceremonies</a> involving a user. The first is <a>Registration</a>, where a <a>scoped credential</a> is created on an <a>authenticator</a>, and associated by the <a>[RP]</a> with the present user's account (the account may already exist or may be created at this time). The second is <a>Authentication</a>, where a <a>[RP]</a> is presented a <em><a>WebAuthn Assertion</a></em> proving the present user is the same as the user who registered the <a>scoped credential</a> associated with the user account. Functionally, the <a>Web Authentication API</a> comprises two methods (along with associated data structures): <a>makeCredential()</a> and <a>getAssertion()</a>. The former is used during <a>Registration</a> and the latter during <a>Authentication</a>.
```

Comment:
ah the key portion being "the presence and consent of the user" -- incorporated. thx.